src/mobile-token/abtClientLogger.ts

```diff
@@ -10,7 +10,7 @@ export const localLogger: ClientConfig['localLogger'] = {
   },
   warn: (msg, err, metadata?) => {
     const onError = toOnErrorCallback('warning', msg, metadata);
-    Bugsnag.notify(err, onError);
+    if (err) Bugsnag.notify(err, onError);
```

Comment:
A fix for https://app.bugsnag.com/atb-as/mittatb/errors/66600166df36580008bb5349?filters[user.id]=57f8e086-e0c7-49ea-b6d9-974838db6cb0&event_id=667bcf1f00efc9de65c30000

It uses the same check as in `error`.

Reply:
So we no longer want to notify every warn event to Bugsnag?

Reply:
What kind of warnings is this? Do you have any examples?

Reply:
If error is undefined, we got a wrong error with no warning info.

Reply:
Aha. That one